ddpui/ddpairbyte/airbytehelpers.py

```diff
@@ -340,12 +398,24 @@ def get_one_connection(org: Org, connection_id: str):
     if dataflow_orgtask is None:
```

Comment:
in the OrgTask query for this connection, don't we need to specify that we want the sync task

Reply:
Yes you are correct. Thanks for the catching that.